src/components/ViewWrapper.vue

```diff
@@ -0,0 +1,387 @@
+<template>
+  <div>
+    <v-container class="view-container my-8 py-0">
+      <v-row>
+        <v-col cols="9" class="left-side">
+          <slot></slot>
+        </v-col>
```

Comment:
The ViewWrapper takes a significant chunk of the template from App.vue.  Here we're using a `<slot>` to allow the views to add back their own content with the minimum of disruption.